src/main/java/com/tenduke/events/api/model/object/LicenseChecked.java

```diff
@@ -0,0 +1,21 @@
+package com.tenduke.events.api.model.object;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.tenduke.events.api.model.data.ErrorInfoFields;
+import com.tenduke.events.api.model.data.LicenseConsumptionFields;
+import com.tenduke.events.api.model.data.RequestFields;
+import com.tenduke.events.api.model.data.TimeFields;
+import com.tenduke.events.api.model.data.UserIdFields;
+
+/**
+ * Event occurring when license has been checked.
+ * @author roope
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public interface LicenseChecked extends
```

Comment:
`LicenseChecked` and `LicenseCheckFailed` look exactly the same here, and both have `ErrorInfoFields`. It would be logical to combine the two, so that only `LicenseChecked` is left. Failure case would be communicated by setting the error fields.